### ISSUE TITLE
When I call the PRINT_FP (11747) emulator hangs in an infinite loop.

### ISSUE THREAD

**Issue by igoninru** [Dotneteer/spectnetide#152]
When I call the PRINT_FP procedure (11747) to display the number on the screen, my emulator hangs in an infinite loop. If you export this code to TAP, and then run it on an external emulator, such as Spectaculator or UnrealSpeccy, then this code will work as expected and will print 12345 on the screen.

```
STACK_BC:
	.equ #2D2B ; 11563
PRINT_FP:
	.equ #2DE3 ; 11747

Start:
    .model Spectrum48
	.org #8000
	ld bc, 12345
	call STACK_BC
	call PRINT_FP ; <- infinite loop inside
	ret
```

**Comment by Dotneteer** [comment]
@igoninru, thanks for signing this issue! I will investigate it and fix it.

**Comment by Dotneteer** [comment]
@igoninru,  I have found the cause of this faulty behavior, now I'm working on fix it.

**Comment by igoninru** [comment]
@Dotneteer Thank you!

**Comment by Dotneteer** [comment]
@igoninru, the issue is fixed and released (v1.19.4)

**_I've just recognized that you gave this project the 100th star!!! Thank you for your support!_**

**There's a folder named `.SpectNetIde` within your ZX Spectrum project folder. Erase this folder with its contents, as it may store the startup state of your virtual machine, The new 1.19.4 version will automatically re-create this folder when you run the Z80 code.**

Please note, that you should not automatically close your code with `RET`. You can read more details here: https://dotneteer.github.io/spectnetide/documents/pragmas#the-xent-pragma 

I used this code to test `PRINT_FP:

```
STACK_BC:
	.equ #2D2B ; 11563
PRINT_FP:
	.equ #2DE3 ; 11747
OPEN_CH:
	.equ #1601 ; 5633
LAST_K:
	.equ #5C08 ; 23560
MAIN_EX:
	.equ #12A9 ; 4777

Start:
    .model Spectrum48
	.org #8000
	ld a,2        ; upper screen
	call OPEN_CH  ; open the channel
	ld bc,12345
	call STACK_BC
	call PRINT_FP 
	jp MAIN_EX    ; Jump back to the MAIN_EX point
```

_Description of the fix_: The virtual machine engine created a startup file that stored the state when the CPU had reached the $12a9 address after restart. Unfortunately it was a state that had not initialized the FP stack, and that caused an infinite loop in your code. I changed the startup address to $12ac to resolve this issue, and this change carries out the desired effect.

**Comment by igoninru** [comment]
@Dotneteer Excellent! Thank you very much for the prompt correction of this error!
